src/main/java/com/powsybl/nad/NetworkAreaDiagram.java

```diff
@@ -84,6 +89,8 @@ public void draw(Path svgFile, SvgParameters svgParameters, LayoutParameters lay
         Objects.requireNonNull(layoutFactory);
         Objects.requireNonNull(idProvider);
 
+        LOGGER.info("Version: {}", new PowsyblNetworkAreaDiagramVersion());
```

Comment:
I would not write it here. That would be done by for instance network-area-diagram-server at startup so just one time.